include/common.sh

```diff
@@ -199,7 +199,7 @@ deleteDuplicateEntries()
 {
   value="\$1"
   delim="\$2"
-  echo "\${value}\${delim}" | awk -v RS="\${delim}" '!(\$0 in a) {a[\$0]; printf("%s%s", col, \$0); col=RS; }' | sed "s/\${delim}$//"
+  echo "\${value}\${delim}" | awk -v RS="\${delim}" '!(\$0 in a) {a[\$0]; printf("%s%s", col, \$0); col=RS; }' | /bin/sed "s/\${delim}$//"
```

Comment:
should this be zossed ?

Reply:
This line is part of the generated zopen-config file.  zopen-config doesn't pull in the common.sh script where the zossed wrapper around /bin/sed is defined. As such, unless we include that wrapper in zopen-config, hardcoding to /bin/sed seems the simplest solution.